modelopt/torch/nas/plugins/megatron_hooks/__init__.py

```diff
@@ -13,20 +13,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""ONNX export utilities."""
+"""Megatron hooks module for NAS plugin."""
+
+from .megatron_hooks import (
+    IndependentChannelContributionHook,
+    IterativeChannelContributionHook,
+    MegatronL2NormHook,
+)
+from .megatron_hooks_analysis import evaluate_importance_scores
 
 __all__ = [
-    "FP8QuantExporter",
-    "INT4QuantExporter",
-    "INT8QuantExporter",
-    "MXFP8QuantExporter",
-    "NVFP4QuantExporter",
-    "ONNXQuantExporter",
+    "IndependentChannelContributionHook",
+    "IterativeChannelContributionHook",
+    "MegatronL2NormHook",
+    "evaluate_importance_scores",
 ]
```

Comment:
Can you follow similar style as other modules: `__all__ = [...]` in individual files and `from .file import *` in `__init__.py`

```suggestion
from .megatron_hooks import *
from .megatron_hooks_analysis import *
```